### PR TITLE
[DE] Add `Music theory/Tempo`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
- ~~In case this gets not reviewed by another native, figure out a kind of placeholder for `strobing` to make the text at least readable. Currently, the two affected places are put in comments (directly in the article).~~
- ~~To add to this, one could also add a comment linking to this PR, so that a review at a later point can pick up the points I explained below in the review comments.~~

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Music_theory/Tempo/de.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# Tempo
+
+*Siehe auch: [Timing](/wiki/Beatmapping/Timing)*
+
+::: Infobox
+![Screenshot der BPM-Anzeige auf der Beatmap-Informationsseite](img/beatmap-info-DE.png "Das Tempo des Songs (181) ist in der oberen Ecke der Beatmap-Informationsseite sichtbar")
+:::
+
+Das **Tempo** bezieht sich auf die Geschwindigkeit des Songs. Normalerweise wird das Tempo in **Beats pro Minute** (***BPM***) gemessen, also die Anzahl an [ganzen Noten](/wiki/Music_theory/Beat) in einer Minute des Songs. Beispielsweise bedeutet ein Tempo von 60 Beats pro Minute ein Beat pro Sekunde, während ein Tempo von 120 Beats pro Minute doppelt so schnell ist und zwei Beats je Sekunde entspricht. Das Tempo wirkt sich auf viele Aspekte des Spiels aus, wie zum Beispiel die [Slider-Geschwindigkeit](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) oder die [Kiai-Zeit](/wiki/Gameplay/Kiai_time) <!--strobing-->. Beatmap-Informationsseiten geben das Tempo neben einem [Metronom](https://de.wikipedia.org/wiki/Metronom) an, ein von Musikern verwendetes Gerät, das "durch akustische Impulse in gleichmäßigen Zeitintervallen ein Tempo vorgibt"[^metronom].
```
Comment (The-Last-Cookie) [inline wiki/Music_theory/Tempo/de.md:17]: I honestly don't know how to handle `strobing`/`strobing`. The closest thing I could think of was `ruckeln`/`abgehackt`.

See [Jerkiness](https://en.wikipedia.org/wiki/Strobbing).

Comment (The-Last-Cookie) [inline wiki/Music_theory/Tempo/de.md:17]: I wanted to go without any citation but I didn't like what else I had in mind, so I went by citing the Wikipedia article.

*Maybe* `das durch gleichmäßige Impulse beim Spielen unterstützt` is an option too.

File: wiki/Music_theory/Tempo/de.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Gameplay
+
+In osu! richten sich viele Spielelemente nach dem Tempo des aktuellen Songs:
+
+- die Spielgeschwindigkeit, da Songs mit höherem Tempo kürzere [Beats](/wiki/Music_theory/Beat) haben
+- die [Slider-Geschwindigkeit](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) in [osu!](/wiki/Game_mode/osu!) und die standardmäßige Scrollgeschwindigkeit in [osu!mania](/wiki/Game_mode/osu!mania)
+- die Frequenz des [Kiai-Zeit](/wiki/Gameplay/Kiai_time) <!--strobbing-->
```
[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/Music_theory/Tempo/de.md:25]:
```suggestion
- die Geschwindigkeit des Aufleuchtens von Hit-Objekten während der [Kiai-Zeit](/wiki/Gameplay/Kiai_time)
```

File: wiki/Music_theory/Tempo/de.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (FreezyLemon) [reply wiki/Music_theory/Tempo/de.md:17]: Both are fine IMO

File: wiki/Music_theory/Tempo/de.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+:::
+
+Das **Tempo** bezieht sich auf die Geschwindigkeit des Songs. Normalerweise wird das Tempo in **Beats pro Minute** (***BPM***) gemessen, also die Anzahl an [ganzen Noten](/wiki/Music_theory/Beat) in einer Minute des Songs. Beispielsweise bedeutet ein Tempo von 60 Beats pro Minute ein Beat pro Sekunde, während ein Tempo von 120 Beats pro Minute doppelt so schnell ist und zwei Beats je Sekunde entspricht. Das Tempo wirkt sich auf viele Aspekte des Spiels aus, wie zum Beispiel die [Slider-Geschwindigkeit](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) oder die [Kiai-Zeit](/wiki/Gameplay/Kiai_time) <!--strobing-->. Beatmap-Informationsseiten geben das Tempo neben einem [Metronom](https://de.wikipedia.org/wiki/Metronom) an, ein von Musikern verwendetes Gerät, das "durch akustische Impulse in gleichmäßigen Zeitintervallen ein Tempo vorgibt"[^metronom].
+
+## Gameplay
+
+In osu! richten sich viele Spielelemente nach dem Tempo des aktuellen Songs:
+
+- die Spielgeschwindigkeit, da Songs mit höherem Tempo kürzere [Beats](/wiki/Music_theory/Beat) haben
```
[the remark this answers is not in the document]
Comment (FreezyLemon) [reply wiki/Music_theory/Tempo/de.md:23]: The texterclub article recommends using whichever case you prefer for enumerations. The SE answers *do* suggest using lowercase.

I guess there's no real consensus. A quick search through German wiki articles reveals that uppercase is preferred in these cases, but lowercase is also regularly used. I guess it's up to you.

File: wiki/Music_theory/Tempo/de.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (FreezyLemon) [reply wiki/Music_theory/Tempo/de.md:25]: LGTM

File: wiki/Music_theory/Tempo/de.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/Music_theory/Tempo/de.md:23]: I go with what I have now. Recently, I saw an article in `Rules` which has bullet points with upper case in the beginning. I will probably fix this to be more consistent.

**Review by TicClick — APPROVED** [review]

**Pull request merged**